RecoTracker/MkFitCore/src/KalmanUtilsMPlex.cc

```diff
@@ -0,0 +1,917 @@
+#include "KalmanUtilsMPlex.h"
+#include "PropagationMPlex.h"
+
+//#define DEBUG
+#include "Debug.h"
+
+#include "KalmanUtilsMPlex.icc"
```

Comment:
Is it auto-generetede? If so, could you please specify it at the beginning of the `*.icc`?

Reply:
it is not auto-generated (in a sense that there is no script that can make it from nothing).
You can see the development history in https://github.com/trackreco/mkFit/commits/devel/mkFit/KalmanUtilsMPlex.icc